src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -180,6 +182,14 @@ const SettingsModalStackNavigator = createModalStackNavigator([
         Component: SettingsPaymentsPage,
         name: 'Settings_Payments',
     },
+    {
+        Component: TransferBalancePage,
+        name: 'Settings_Transfer_Balance',
+    },
+    {
+        Component: ChooseTransferAccountPage,
+        name: 'Settings_Choose_Transfer_Account',
```

Comment:
I feel we could make it clearer that this is related to the Payments page and that would be a good pattern to follow.

We have `SETTINGS_PAYMENTS_CHOOSE_TRANSFER_ACCOUNT` but the name for this is just `Settings_Choose_Transfer_Account` and should probably be changed to `Settings_Payments_Choose_Transfer_Account`

Reply:
Ok. I was thinking the same.